java/sage/media/sub/VobSubSubtitleHandler.java

```diff
@@ -46,24 +54,25 @@ public void loadSubtitlesFromFiles(sage.MediaFile sourceFile)
     try
     {
       subFile = getLoadableSubtitleFile(sourceFile, new java.io.File(subs[0].getPath()));
-      inStream = sage.IOUtils.openReaderDetectCharset(subFile, sage.Sage.BYTE_CHARSET);
+      inStream = sage.IOUtils.openReaderDetectCharset(subFile, sage.Sage.BYTE_CHARSET, sourceFile.isLocalFile());
```

Comment:
Looks like you forgot to update this one to not use getLoadableSubtitleFile (and also set the subFile path the way you were doing in the other classes).